README.md

```diff
@@ -39,6 +39,24 @@ You can also incrementally add new warnings to the ignorefile as you encounter t
 RECORD_DEPRECATIONS=1 rspec path/to/my/failing/spec.rb
 ```
 
+## Deprecating Attributes
+
```

Comment:
Here's my first pass at a README entry! I like how @smudge gave a real-world example on why one might want to record deprecations above ex: `When testing on a new Rails version for the first time, you will undoubtedly encounter many new warnings.` - curious if @pat-whitrock or @smudge  have some suggestions on adding a real-world example to this? Would that be helpful to add?